node_test/integration_test.go

```diff
@@ -96,7 +76,7 @@ func RunIntegrationTestCase(tc TestCase, t *testing.T) {
 		intermediaries := make([]node.Node, 0)
 		bootPeers := make([]string, 0)
 		for _, intermediary := range tc.Participants[2:] {
```

Comment:
This seems pretty brittle since we have to put the intermediaries at the end of the Participants array. Would it be cleaner to add another field in our `TestCase` struct to separate the Intermediaries like this?
```
TestCase{
  ...
  Participants: []TestParticipant{
    {StoreType: DurableStore, Actor: testactors.Alice},
    {StoreType: DurableStore, Actor: testactors.Bob},
  },
  Intermediaries: []TestParticpant{
    {StoreType: DurableStore, Actor: testactors.Irene},
    {StoreType: DurableStore, Actor: testactors.Ivan},
  },
}
```
Then instead of having to know where the intermediaries fall in the `tc.Participants` array, we could just loop over the separate `tc.Intermediaries` array:
```
		for _, intermediary := range tc.Intermediaries {
```

Reply:
I know this isn't really part of this PR, just caught my eye when reviewing

Reply:
That's a good idea! Although we do have some test case validation based on the [amount of participants](https://github.com/statechannels/go-nitro/blob/0c9ccee6ddaddc0ea977657c7494024d4702a147/node_test/types_test.go#L68) so we could also consider a flag on the `TestParticipant` that can be set instead of a seperate slice.